stable/app/README.md

```diff
@@ -40,57 +40,61 @@ The command removes all the Kubernetes components associated with the chart and
-| service.port.protocol                 | string | `"TCP"`                                                                                       |                                                                      |
-| service.port.targetPort               | string | `"tcp"`                                                                                       |                                                                      |
-| service.type                          | string | `"ClusterIP"`                                                                                 |                                                                      |
-| tolerations                           | list   | `[]`                                                                                          |                                                                      |
-| migration.enabled                     | bool   | `false`                                                                                       |                                                                      |
-| migration.command                     | list   | `[]`                                                                                       |                                                                      |
-| migration.args                        | list   | `[]`                
-|                                                                      |
-| telegraf.enabled                      | bool   | `false`                
-|                                                                      |
-| telegraf.command                      | list   | `["telegraf"]`                
-|                                                                      |
-| telegraf.args                         | list   | `[]`                
-|                                                                      |
-| telegraf.containerPort                | int    | `8125`                
-|                                                                      |
-| telegraf.protocol                     | string | `UDP`                
-|                                                                      |
-| telegraf.config                       | string | `""`                | telegraf config file content |
+| cron.enabled           | bool | `false`                                                                                       |  |
+| cron.job.entryPoint    | string | `""`                                                                                          |  |
+| cron.job.name          | string | `""`                                                                                  |  |
+| cron.job.schedule      | string | `"0 0 * * *"`                                                                                 |  |
```

Comment:
Doesn't seem to match the changes made in templates and values.

Reply:
fixed and updated the docs